astropy/tests/helper.py

```diff
@@ -222,9 +222,11 @@ def treat_deprecations_as_exceptions():
     for module in list(sys.modules.values()):
         # We don't want to deal with six.MovedModules, only "real"
         # modules. FIXME: we no more use six, this should be useless ?
```

Comment:
LoL this comment is so outdated but it is out of scope here.

Reply:
Separate PR to remove the comment?